zeekpkg/manager.py

```diff
@@ -2239,6 +2283,16 @@ def _stage(self, package, version, clone,
 
         raw_metadata = _get_package_metadata(raw_metadata_parser)
 
+        # Ensure any listed executables exist as advertised. Do this first so
+        # that we don't build anything if it fails.
```

Comment:
Don't understand the reason for this check happening before execution of `build_command`.  Can that not end up producing the desired executable to install ?  The new docs showed an example with `build/a.out` which I took to mean as implied/intended support for `build_command` producing the executable.

Reply:
Oh, you're absolutely right. That ordering doesn't make sense, the check needs to move after the build. I'll do a quick check and then another PR later.